src/combobox/styles/comboBox.m.css

```diff
@@ -31,27 +31,26 @@
 .results {
 	position: absolute;
 	width: 100%;
-	max-height: var(--select-menu-height);
+	max-height: 400px;
```

Comment:
Re: the comment above, we should make a ticket to variable-ize the base theme for maintainability and consistency before landing this so we don't lose track of that work.